### DISCUSSION TITLE
영어로된 주석 링크

### DISCUSSION THREAD

**Discussion by huihyeon06** [MSG-Mirim-Study-Group/Your-Planet#2] [Q&A] [answered]
주세요

**Comment by haeun-noh** [comment, marked as answer]
https://learn.microsoft.com/ko-kr/dotnet/csharp/language-reference/xmldoc/recommended-tags

**Comment by haeun-noh** [comment]
https://learn.microsoft.com/ko-kr/dotnet/csharp/language-reference/xmldoc/
